TLDR Adapter config table (we'll be speaking with Benoit about this PR) (#7163)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/databricks-configs.md

id: "databricks-configs"
---

> | Configuration      | Description    | 

Reviewer: sugg having a nice intro sentence to this so users know what the table is?

|--------------------|----------------|
|[Configuring tables](/reference/resource-configs/databricks-configs#configuring-tables)|See table for more information on the optional configs you can use when materializing a model as table.|
|[Incremental models](/reference/resource-configs/databricks-configs#incremental-models-1)|The `incremental_strategy` config determines how dbt builds incremental models. The dbt-databricks plugin supports the following incremental materialization strategies &mdash; `append`, `insert_overwrite`, `merge`, [`microbatch`](/docs/build/incremental-microbatch) and `replace_where`.|

> |[Selecting compute per model](/reference/resource-configs/databricks-configs#selecting-compute-per-model)|From v1.7.2, you can assign which compute resource to use on a per-model basis. |

Reviewer: do we want to specify this can be done for sql and python models?

> |[`persist_docs`](/reference/resource-configs/databricks-configs#persisting-model-descriptions)|When the `persist_docs` is configured correctly, model descriptions will appear in the `Comment` field of `describe [table] extended` or `show table extended in [database] like '*'`.|

Reviewer: sugg active voice
```suggestion
|[`persist_docs`](/reference/resource-configs/databricks-configs#persisting-model-descriptions) | Configure `persist_docs` to add model descriptions to the `Comment` field shown in `describe [table] extended` or `show table extended in [database] like '*'`. |
```

> |[Default file format configurations](/reference/resource-configs/databricks-configs#default-file-format-configurations)|Use the Delta or Hudi file format as the default file format to use advanced incremental strategies features.|

Reviewer: do we want to explicitly share the config?
```suggestion
|[Default file format configurations](/reference/resource-configs/databricks-configs#default-file-format-configurations)| Use the Delta or Hudi file format (`file_format`) as the default file format to use advanced incremental strategies features.|
```

> |[Materialized views and streaming tables](/reference/resource-configs/databricks-configs#materialized-views-and-streaming-tables-1)|Alternatives to incremental tables that are powered by Delta Live Tables. A workspace that is enabled for Unity Catalog and serverless SQL Warehouses will be needed.|

Reviewer: sugg active voice
```suggestion
|[Materialized views and streaming tables](/reference/resource-configs/databricks-configs#materialized-views-and-streaming-tables-1) | Use Delta Live Tables as an alternative to incremental tables. To do this, enable Unity Catalog and serverless SQL Warehouses in your workspace. |
```
